app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,12 @@
+class ProjectsController < ApplicationController
+
+  def show
+    @project = Project.find(params[:id])
+  end
+
+
+  private
+  def project_params
```

Comment:
It doesn't seem like you are using these anywhere. I would only build what you need at the time.